File: backend/src/main/java/com/verisure/practice/presentation/dto/RoundDTO.java
package com.verisure.practice.presentation.dto;

import java.util.List;

import com.verisure.practice.domain.MatchPair;

public class RoundDTO {
    private int round;
    private List<MatchPair> pairs;
    
    public RoundDTO(int round, List<MatchPair> pairs) {
        this.round = round;
        this.pairs = pairs;
    }
    

    public int getRound() {
        return round;
    }
    
    public void setRound(int round) {
        this.round = round;
    }
    
    public List<MatchPair> getPairs() {
        return pairs;
    }
    
    public void setPairs(List<MatchPair> pairs) {
        this.pairs = pairs;
    }

}

File: backend/CiP-03-eWorldCupManager.Domain/Dtos/RoundDTO.cs
using CiP_03_eWorldCupManager.Domain.Models;

namespace CiP_03_eWorldCupManager.Domain.Dtos;

public class RoundDTO
{
    public int Round { get; init; }
    //public IList<MatchPair> Pairs { get; init; }
    public IList<PairDTO> Pairs { get; init; }

    public RoundDTO(int round, IList<PairDTO> pairs)
    {
        Round = round;
        Pairs = pairs;
    }
}
